README.md

```diff
@@ -17,10 +17,11 @@ The first twenty-four examples are being added as an advent calendar leading up
 5. [Aggregating Data](recipes/03/03_group_by.md)
 6. [Sorting Tables](recipes/08/08_order_by.md)
 
-### Aggregating over Time (Time Windows) 
+### Aggregating over Time
 1. [Aggregating Time Series Data](recipes/06/06_group_by_window.md)
 2. [Watermarks](recipes/07/07_watermarks.md)
 3. [Analyzing Sessions in Time Series Data](recipes/09/09_group_by_session_window.md)
+4. [Rolling Aggregations on Time Series Data](recipes/10/10_over.md)
```

Comment:
Are we doing it again here? 😀 Repeating "Time Series Data" over and over when we have "over time" in the section title?

Reply:
I don't know. I am also a bit confused by the Time Series Data as kind of a synonym for tables with a time attribute.